packages/ffe-datepicker-react/src/calendar/MonthPicker.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+
+export default function MonthPicker(props) {
+    const { previousMonthHandler, nextMonthHandler, month } = props;
+
+    function left(evt) {
+        previousMonthHandler(evt);
+    }
+
+    function right(evt) {
+        nextMonthHandler(evt);
+    }
+
+    return (
+        <div className="ffe-calendar__monthpicker">
+            <div className="ffe-calendar__monthpicker-center">{month}</div>
+            <div
+                role="none"
+                onClick={left}
```

Comment:
A div with onClick and no role? I'm not UU expert, but this doesn't sound so good to me.
We had a button with icons in it, seemed alot more sensible.

Reply:
I guess i did this way because I needed to tilt the chevron icon 90 deg. Was not sure how to implement this with button.

Reply:
```
some-selector-bladi-bladi {
            svg {
                transform: rotate(180deg);
            }
}
```

Reply:
Generally speaking. If you use a onClick on a div you would need to provide a role(other then none). You would also need to handle focus, clicking (with style). Probaly i forgot something now but generally its much easier to just use a normal html button and fix the styling.